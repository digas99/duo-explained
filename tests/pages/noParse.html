<!DOCTYPE html>
<html data-duo-theme="dark" dir="ltr" lang="en"><head><title data-react-helmet="true">Duolingo - The world's best way to learn French</title>     </head><body style="--onetrust-banner-height: 0px;"><div class="fs-unmask" id="root" onclick=""><div class="kPqwA _2kkzG"><div class="_3yE3H"><div class="_2G9ED _29gfw"><div class="wqSzE"><div class="_3v4ux"><div class="I-Avc _1zcW8"><button class="_3vGNs _2YF0P _3X5b9" data-test="quit-button"><img class="_3X5b9 _1o-YO" src="./noParse_files/df223d5b9feb8017b323ed21103eb5ac.svg"/></button><div aria-valuemax="1" aria-valuemin="0" aria-valuenow="0.10585733481633375" class="_358Nq" role="progressbar" style="--web-ui_progress-bar-color: rgb(var(--color-owl)); --web-ui_progress-bar-shine-height: 3px; --__internal__progress-bar-height: 16px; --__internal__progress-bar-inner-value: 0%; --__internal__progress-bar-value: 10.59%;"><div class="_2-LJi"><div class="_2fuvy" style="opacity: 1;"><div class="_3Dzp2 _2fuvy"></div><div class="_1EzVJ"></div></div><div class="_3FEy7" style="opacity: 1;"><div class="_2fYtx"></div></div></div></div><div class="_3ww7z"><img src="./noParse_files/0c1a523b4b5882a97e4df162f4b5c58b.svg" style="height: 32px; width: 32px;"/><img src="./noParse_files/8d4c5fcd9cedabfd155d3eda8af269bc.svg"/></div></div></div></div><div class="RMEuZ _1GVfY"><div class="_3GuWo _1cTBC"><div class="_1XNQX _3V6my"><div class="_1fxa4 _1Mopf" data-test="challenge challenge-noParse"><div class="_2n5fx _1JTA4 _3B8G- K3vbJ"><div class="U41Ye"><h1 class="_3EOK0" data-test="challenge-header"><span>Template for no parsing</span></h1></div><div class="BZl0l UjFh4 _3rat3"><div class="_3_UP9 _1dtTU"><div class="_31yjb"><div class="_2qg6J"><div class="" style="width: 100%; height: 100%; transform: scale(1.1) translate(-6.94444%, 0%);"><canvas height="175" style="vertical-align: top; width: 118px; height: 175px;" width="118"></canvas></div></div><div class="_1UOaB _2XSwf _44BHt" style="z-index: 150;"><div class="_1DLP9 _3Hvln"><span><span class="_2bNpZ _1fdKO" dir="ltr"><button class="_2iVKu _3DAip bafGS _2LoNU VzbUl _1saKQ _1AgKJ"><span class="RxQcJ fs-exclude _1OCYa" style="--animated-speaker-icon-color: rgb(var(--color-macaw));"></span></button></span></span><span><span class="_2bNpZ _1fdKO" dir="ltr"><button class="_1dsPf bafGS _2LoNU VzbUl _1saKQ _1AgKJ"><span class="RxQcJ fs-exclude _1OCYa" style="--animated-speaker-icon-color: rgb(var(--color-macaw));"></span></button></span></span></div><div class="Vpm8Y" style="top: calc(50% - 15px); transform: translateY(-50%) rotate(-90deg);"><div class="_1IAp5"></div></div></div></div></div><div class="_2SnPj _32LPr _3hg-V"><label class="H-Fgo _2mrQw RpiVp" dir="ltr"><span lang="fr">Je</span><span lang="fr"> </span><span lang="fr">suis</span><span lang="fr"> </span><span class="_2OBgL"><div class="zlazS">_devant_</div><input autocapitalize="off" autocomplete="off" autocorrect="off" class="YQVzO _3mwuq" data-test="challenge-text-input" dir="ltr" lang="fr" size="1" spellcheck="false" type="text" value=""/></span><span lang="fr"> </span><span lang="fr">le</span><span lang="fr"> </span><span lang="fr">restaurant</span><span lang="fr">.</span></label><div class="_3SM2V _3qBhY" lang="fr"><div class="wQ_9d"><button class="_1Qh5D _36g4N _2YF0P _2yFpW" type="submit"><span class="_1o-YO">↓</span></button></div><div class="wQ_9d"><button class="_1Qh5D _36g4N _2YF0P _2yFpW" type="submit"><span class="_1o-YO">À</span></button></div><div class="wQ_9d"><button class="_1Qh5D _36g4N _2YF0P _2yFpW" type="submit"><span class="_1o-YO">Â</span></button></div><div class="wQ_9d"><button class="_1Qh5D _36g4N _2YF0P _2yFpW" type="submit"><span class="_1o-YO">Æ</span></button></div><div class="wQ_9d"><button class="_1Qh5D _36g4N _2YF0P _2yFpW" type="submit"><span class="_1o-YO">È</span></button></div><div class="wQ_9d"><button class="_1Qh5D _36g4N _2YF0P _2yFpW" type="submit"><span class="_1o-YO">É</span></button></div><div class="wQ_9d"><button class="_1Qh5D _36g4N _2YF0P _2yFpW" type="submit"><span class="_1o-YO">Ê</span></button></div><div class="wQ_9d"><button class="_1Qh5D _36g4N _2YF0P _2yFpW" type="submit"><span class="_1o-YO">Ë</span></button></div><div class="wQ_9d"><button class="_1Qh5D _36g4N _2YF0P _2yFpW" type="submit"><span class="_1o-YO">Î</span></button></div><div class="wQ_9d"><button class="_1Qh5D _36g4N _2YF0P _2yFpW" type="submit"><span class="_1o-YO">Ï</span></button></div><div class="wQ_9d"><button class="_1Qh5D _36g4N _2YF0P _2yFpW" type="submit"><span class="_1o-YO">Ô</span></button></div><div class="wQ_9d"><button class="_1Qh5D _36g4N _2YF0P _2yFpW" type="submit"><span class="_1o-YO">Ù</span></button></div><div class="wQ_9d"><button class="_1Qh5D _36g4N _2YF0P _2yFpW" type="submit"><span class="_1o-YO">Û</span></button></div><div class="wQ_9d"><button class="_1Qh5D _36g4N _2YF0P _2yFpW" type="submit"><span class="_1o-YO">Ü</span></button></div><div class="wQ_9d"><button class="_1Qh5D _36g4N _2YF0P _2yFpW" type="submit"><span class="_1o-YO">Ç</span></button></div><div class="wQ_9d"><button class="_1Qh5D _36g4N _2YF0P _2yFpW" type="submit"><span class="_1o-YO">Œ</span></button></div></div></div></div></div></div></div></div></div><div class="_3GuWo _1cTBC _1QQhE"><div class="_1XNQX"><div class="_3rB4d _1VTif _2HXQ9 d-cgpt-footer" id="session/PlayerFooter"><div class="U8jH3 jHbiF"><div class="_3h0lA"><button class="_1Qh5D _36g4N _2YF0P _2x7Co _3fo6Q" data-test="player-skip"><span class="_1o-YO">Can't listen now</span></button></div><div class="MYehf d-cgpt-explain-button-wrapper"><button aria-disabled="true" class="_1NM1Q _1x5JY _1M9iF _36g4N _2YF0P _3DbUj _38g3s _2oGJR d-cgpt-explain-button d-cgpt-explain-button-disabled" data-hasclick="false" disabled="" id="d-cgpt-explain-button"><span class="_1o-YO">Explain</span></button><button aria-disabled="true" class="_1NM1Q _1x5JY _1M9iF _36g4N _2YF0P _3DbUj _38g3s _2oGJR" data-test="player-next"><span class="_1o-YO">Check</span></button></div></div></div>
<div class="d-cgpt-speech-bubble" style="bottom: 130px; display: none;">
<div>
<textarea placeholder="If you want, add to the query to ChatGPT..."></textarea>
<div class="d-cgpt-speech-bubble-tip"></div>
</div>
</div>
</div></div></div></div></div><div class="Vm8CO _1Fnem" data-test="drawer-backdrop" style="pointer-events: none;"></div></div><div class="fs-unmask" id="overlays" onclick=""><div class="Vm8CO _2zxQ8" style="pointer-events: none;"></div><a class="_1Qh5D _36g4N _2YF0P _3dDzz _1wiIJ" href="https://preview.duolingo.com/help/support-request" rel="noopener" target="_blank" title="Report feedback"><img class="_3QUr-" src="./noParse_files/b23d90ca7ea01eb330158d4939dc4899.svg"/></a><div data-focus-guard="true" style="width: 1px; height: 0px; padding: 0px; overflow: hidden; position: fixed; top: 1px; left: 1px;" tabindex="-1"></div><div data-focus-lock-disabled="disabled"></div><div data-focus-guard="true" style="width: 1px; height: 0px; padding: 0px; overflow: hidden; position: fixed; top: 1px; left: 1px;" tabindex="-1"></div></div><div><div class="grecaptcha-badge" data-style="bottomright" style="width: 256px; height: 60px; display: block; transition: right 0.3s; position: fixed; bottom: 14px; right: -186px; box-shadow: gray 0px 0px 5px; border-radius: 2px; overflow: hidden;"><div class="grecaptcha-logo"></div><div class="grecaptcha-error"></div><textarea class="g-recaptcha-response" id="g-recaptcha-response-100000" name="g-recaptcha-response" style="width: 250px; height: 40px; border: 1px solid rgb(193, 193, 193); margin: 10px 25px; padding: 0px; resize: none; display: none;"></textarea></div></div><div id="onetrust-consent-sdk"><div class="onetrust-pc-dark-filter ot-hide ot-fade-in"></div></div></body></html>