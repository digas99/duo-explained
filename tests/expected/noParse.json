{
	"exercise": "Template for no parsing",
	"type": "noParse",
	"wrapper": {}
}